.clang-tidy

```diff
@@ -1,163 +1,178 @@
-    value:           'true'
-  - key:             modernize-loop-convert.NamingStyle
-    value:           CamelCase
-  - key:             bugprone-suspicious-include.ImplementationFileExtensions
-    value:           'c;cc;cpp;cxx'
-  - key:             bugprone-suspicious-missing-comma.SizeThreshold
-    value:           '5'
-  - key:             bugprone-suspicious-include.HeaderFileExtensions
-    value:           ';h;hh;hpp;hxx'
-  - key:             google-readability-function-size.StatementThreshold
-    value:           '800'
-  - key:             llvm-else-after-return.WarnOnConditionVariables
-    value:           'false'
-  - key:             bugprone-argument-comment.CommentCharacterLiterals
-    value:           '0'
-  - key:             bugprone-argument-comment.CommentIntegerLiterals
-    value:           '0'
-  - key:             bugprone-sizeof-expression.WarnOnSizeOfCompareToConstant
-    value:           'true'
-  - key:             modernize-pass-by-value.IncludeStyle
-    value:           llvm
-  - key:             bugprone-reserved-identifier.AggressiveDependentMemberLookup
-    value:           'false'
-  - key:             bugprone-sizeof-expression.WarnOnSizeOfThis
-    value:           'true'
-  - key:             bugprone-string-constructor.WarnOnLargeLength
-    value:           'true'
-  - key:             bugprone-too-small-loop-variable.MagnitudeBitsUpperLimit
-    value:           '16'
-  - key:             bugprone-argument-comment.CommentFloatLiterals
-    value:           '0'
-  - key:             modernize-use-nullptr.NullMacros
-    value:           'NULL'
-  - key:             bugprone-dangling-handle.HandleClasses
-    value:           'std::basic_string_view;std::experimental::basic_string_view'
-  - key:             bugprone-dynamic-static-initializers.HeaderFileExtensions
-    value:           ';h;hh;hpp;hxx'
-  - key:             bugprone-suspicious-enum-usage.StrictMode
-    value:           'false'
-  - key:             bugprone-implicit-widening-of-multiplication-result.IncludeStyle
-    value:           llvm
-  - key:             bugprone-suspicious-missing-comma.MaxConcatenatedTokens
-    value:           '5'
-  - key:             bugprone-implicit-widening-of-multiplication-result.UseCXXHeadersInCppSources
-    value:           'true'
-  - key:             llvm-else-after-return.WarnOnUnfixable
-    value:           'false'
-  - key:             bugprone-not-null-terminated-result.WantToUseSafeFunctions
-    value:           'true'
-...
+  - key: readability-identifier-naming.ClassCase
+    value: CamelCase
+  - key: readability-identifier-naming.StructCase
+    value: CamelCase
+  - key: readability-identifier-naming.FunctionCase
+    value: camelBack
+  - key: readability-identifier-naming.ClassMemberCase
+    value: camelBack
+  - key: readability-identifier-naming.ClassMethodCase
+    value: camelBack
+  - key: readability-identifier-naming.VariableCase
+    value: camelBack
```

Comment:
having ClassMethodCase and VariableCase the same is confusing (I just tried it). How about `lower_case` for VariableCase?